internal/alb/sg/namer.go

```diff
@@ -0,0 +1,26 @@
+package sg
+
+import (
+	"fmt"
+)
+
+const (
+	lbSGNamePtn       = "%s"
+	instanceSGNamePtn = "instance-%s"
+)
+
+// Namer can name securityGroup related resources.
+type Namer interface {
+	NameLbSG(loadBalancerID string) string
+	NameInstanceSG(loadBalancerID string) string
+}
+
+type namer struct{}
+
+func (namer *namer) NameLbSG(loadBalancerID string) string {
+	return fmt.Sprintf(lbSGNamePtn, loadBalancerID)
```

Comment:
NIT: feels its completely fine to inline `lbSGNamePtn` and `instanceSGNamePtn` to where it is used inside method if only one method uses it. It has the benefit that, during refactoring, the constant will be moved along with the method instead of lingering around.

Reply:
yeah..this part can be inlined in the association that created it.
I created a separate component here because the naming(and also tagging?) of AWS resources are important and should have consist rule/patterns.
The ultimate goal is to have a simple component generating names/tag for all aws resources we created. I think it's better than distribute them around different modules(hard to apply a consist rules to them)

Reply:
misunderstanded the comment, will move the constant into the method